sim/simx/decode.cpp

```diff
@@ -405,6 +406,16 @@ static const char* op_string(const Instr &instr) {
     default:
       std::abort();
     }
+  
+  case Opcode::TCU:
+    switch(func3)
+    {
+      case 0: return "ML"; //
+      case 1: return "MS"; //
+      case 2: return "MATMUL";
```

Comment:
Either add a comment here which would be great or remove the empty comments.

Reply:
Fixed.